stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/service/sync/PushMessageSyncHandler.kt

```diff
@@ -79,22 +69,16 @@ public class PushMessageSyncHandler(private val service: Service) {
         startForegroundExecution(notificationConfig.smallIcon)
 
         val cid: String = firebaseMessageParser.parse(message).let { "${it.channelType}:${it.channelId}" }
-        if (ChatDomain.isInitialized && ChatClient.isInitialized) {
-            logger.logD("Starting the sync")
-            performSync(ChatDomain.instance(), cid, ChatClient.instance(), message)
-        } else {
-            val syncConfig = syncModule.encryptedBackgroundSyncConfigStore.get()
-            syncConfig?.let {
-                val config = it
-                val user = User(id = config.userId)
-                val token = config.userToken
-                val client = initClient(service, user, token, config.apiKey)
-                val domain = initDomain(user, client)
+        val syncConfig = syncModule.encryptedBackgroundSyncConfigStore.get()
+        syncConfig?.let {
+            val config = it
+            val user = User(id = config.userId)
+            val token = config.userToken
+            ChatClient.instance().setUserWithoutConnecting(user, token)
```

Comment:
This method actually does something a bit different. Is sets the user, but only if  it was not set before. So we can safely call the method here.